### PR TITLE
Removed all references to the utility classes and referenced the appr…

### PR DESCRIPTION
**Pull request by rosecers**
Removed all references to the utility classes and referenced the appropriate sklearn classes.
Changed narrative to explain new syntax where necessary.
Remove utility reference of MDS, because as written it is redundant with PCA and holds no operational value.

Outstanding changes: 
 - [ ] reference skcosmo kernel scalers, when ready
 - [ ] Check for naming mismatches with new sklearn referencing

@PKGuo I have "sklearnified" the first notebook as an example. Please do a PR review of it, noting the changes. If there are any changes that don't make sense or seem not the best choice, add comments. Please read through carefully so that giving the same treatment to the other notebooks is easier.

### CODE REVIEW

File: notebooks/1_LinearMethods.ipynb [hunk 1]
```diff
@@ -870,7 +872,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "coeff_deter = np.array([lr.statistics(X_test, Y_test)['Coefficient of Determination<br>($R^2$)'] for lr in lrs])"
+    "ell = np.array([np.linalg.norm(Y_test - lr.predict(X_test)) ** 2.0 / np.linalg.norm(Y_test) **2.0 for lr in lrs])"
```
Comment (PKGuo) [inline notebooks/1_LinearMethods.ipynb:875]: Why replace coeff_deter (coefficient of determination) with ell (empirical loss)?

Comment (PKGuo) [reply notebooks/1_LinearMethods.ipynb:875]: Is it because there is no corresponding method for lr.statistics in sklearn?

File: notebooks/1_LinearMethods.ipynb [hunk 2]
```diff
@@ -1068,77 +1042,20 @@
...
-    "    pca2 = PCA(n_PC=n)\n",
-    "    pca2.fit(X_train)\n",
-    "\n",
-    "    T_train2 = pca2.transform(X_train)\n",
-    "    T_test2 = pca2.transform(X_test)\n",
-    "\n",
-    "    PTY2 = np.matmul(T_train2.T, T_train2) + best_regularization*np.eye(T_train2.shape[1])\n",
-    "    PTY2 = np.linalg.pinv(PTY2)\n",
-    "    PTY2 = np.matmul(PTY2, T_train2.T)\n",
-    "    PTY2 = np.matmul(PTY2, Y_train)\n",
```
Comment (PKGuo) [inline notebooks/1_LinearMethods.ipynb:1086]: Maybe we can keep this part and move it below line 955.  This will help readers understand "including more components gives a better prediction of the properties."

File: notebooks/1_LinearMethods.ipynb [hunk 3]
```diff
@@ -1156,17 +1074,21 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Calling `mds.statistics(X)` outputs the statistics of the projection of $\\mathbf{X}$."
+    "## LR\n",
+    "\n",
+    "Unlike PCA, which takes the number of PCA components in the constructor, the linear regression class takes the regularization as a parameter.\n",
```
Comment (PKGuo) [inline notebooks/1_LinearMethods.ipynb:1089]: I don't quite understand this sentence. According to sklearn's documentation, both **n_components** and **alpha** are **parameters**. What do you want to emphasize here?
image
image

File: notebooks/1_LinearMethods.ipynb [hunk 4]
```diff
@@ -1175,18 +1097,8 @@
...
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## LR\n",
-    "\n",
-    "Unlike PCA, which takes the number of PCA components in the constructor, the linear regression class takes the regularization as a parameter."
+    "from sklearn.linear_model import LinearRegression as LR\n",
```
Comment (PKGuo) [inline notebooks/1_LinearMethods.ipynb:1100]: I think it might be unnecessary to import LinearRegregation because we've never used it.

**Review by PKGuo — CHANGES_REQUESTED** [review]
Thanks for your example. I will "sklearnified" other notebooks like this.

File: notebooks/1_LinearMethods.ipynb (hunk 1 above)
Comment (rosecers) [reply notebooks/1_LinearMethods.ipynb:875]: Correct! It's easier to use the relative error here

File: notebooks/1_LinearMethods.ipynb (hunk 2 above)
Comment (rosecers) [reply notebooks/1_LinearMethods.ipynb:1086]: What's your case for keeping it? I don't see a strong reason to

File: notebooks/1_LinearMethods.ipynb (hunk 3 above)
Comment (rosecers) [reply notebooks/1_LinearMethods.ipynb:1089]: No, PCA doesn't take alpha as a parameter and Ridge doesn't take n_components as a parameter.

File: notebooks/1_LinearMethods.ipynb (hunk 4 above)
Comment (rosecers) [reply notebooks/1_LinearMethods.ipynb:1100]: This is to show the two sklearn imports

File: notebooks/1_LinearMethods.ipynb (hunk 2 above)
Comment (PKGuo) [reply notebooks/1_LinearMethods.ipynb:1086]: I just think it is better to keep it than to delete it. What is the reason for deleting it?

File: notebooks/1_LinearMethods.ipynb (hunk 3 above)
Comment (PKGuo) [reply notebooks/1_LinearMethods.ipynb:1089]: Oh, yes. I am being silly.

File: notebooks/1_LinearMethods.ipynb (hunk 4 above)
Comment (PKGuo) [reply notebooks/1_LinearMethods.ipynb:1100]: Okay.

File: notebooks/1_LinearMethods.ipynb (hunk 2 above)
Comment (rosecers) [reply notebooks/1_LinearMethods.ipynb:1086]: So PCR gets better with larger numbers of PCs, but making it the focus here detracts from the overall focus of the notebook. Additionally, sklearn doesn't have a nice, easy PCR, because [they favor PLS and suggest just using LR on PCA for PCR](https://scikit-learn.org/dev/auto_examples/cross_decomposition/plot_pcr_vs_pls.html). 

I think there are far more reasons to delete it than "it is better to keep it".

**Comment by rosecers** [comment]
@PKGuo do you approve the PR?

File: notebooks/1_LinearMethods.ipynb (hunk 2 above)
Comment (PKGuo) [reply notebooks/1_LinearMethods.ipynb:1086]: Okay, I agree.

**Review by PKGuo — APPROVED** [review]
